root/layout/components/sidebar/AttendanceLinks.js

```diff
@@ -20,33 +20,37 @@ type Props = {|
 |};
 
 const AttendanceLinks = ({$c, event}: Props) => {
-  const allCollections = $c.stash.all_collections;
-  if (!$c.user_exists || !allCollections) {
+  const numberOfCollections = $c.stash.number_of_collections || 0;
+  if (!$c.user_exists) {
     return null;
   }
   return (
-    <>
-      <h2 className="attendance">
-        {l('Attendance')}
-      </h2>
-      <CollectionList
-        addText={l('Add to a new list')}
-        entity={event}
-        noneText={l('You have no attendance lists!')}
-        usersLink={
-          <EntityLink
-            content={texp.ln(
-              'Found in {num} attendance list',
-              'Found in {num} attendance lists',
-              allCollections.length,
-              {num: allCollections.length},
-            )}
-            entity={event}
-            subPath="attendance"
-          />
-        }
-      />
-    </>
+    <CollectionList
+      addCollectionText={l('Add to a new list')}
+      collaborativeCollections={$c.stash.collaborative_collections}
+      collaborativeCollectionsHeader={l('Collaborative lists')}
+      collaborativeCollectionsNoneText={
+        l('Not collaborating on any attendance lists!')
+      }
+      entity={event}
+      header={l('Attendance')}
+      ownCollections={$c.stash.own_collections}
+      ownCollectionsHeader={l('My attendance lists')}
```

Comment:
When I think about it, it's weird that we assume every event collection should be an attendance list. I think having the default "Attending," "Maybe attending" collections is enough to satisfy that use case, and doesn't mean we should label all uses as "Attendance lists." Especially with collaborative collections, I can imagine some like "events where band X performed with lineup Y" or "highest grossing events" or "highest attended events" (granted that last one kind of is an attendance list of sorts :)). Or any collection where the events share some attribute about the artists/performers that might be interesting and either hard to store in MB or hard to search for.

Reply:
I've always thought it's a bit weird, so I'm more than happy to consolidate all of these as just collections :)